Client config local repo warning #13107 keichwa (#681)
uyuni-project/uyuni-docs: modules/client-configuration/pages/repositories.adoc

You can configure local repositories on Salt clients, to provide packages that are not supplied by {productname} channels.

[WARNING]
====

> In general, client systems should have local repositories configured.

Reviewer: A "not" is missing

```suggestion
In general, client systems should not have local repositories configured.
```

> Local repositories can cause management because you otfen do not know what is available on the client system and you can get packages installed you did not expect.

Reviewer: ```suggestion
Local repositories can cause management troubles because you do not know what is available on the client system and you can get packages installed you did not expect.
```

> In general, client systems should not have local repositories configured.
> Local repositories can cause management troubles because you do not know what is available on the client system and you can get packages installed you did not expect.

Reviewer 2: ```suggestion
In most cases, client systems do not require local repositories.
Local repositories can lead to problems knowing which packages are available on the client.
This can lead to installing unexpected packages.
```

====

> Local repositories get disabled during onboarding.

Reviewer 2: ```suggestion
Local repositories are disabled during onboarding.
```
